kaska/inference_runner.py

```diff
@@ -175,8 +183,14 @@ def process_tile(the_chunk, config):
             config.output_folder,
             chunk=hex(chunk_no),
         )
-        parameter_names, parameter_data = kaska.run_retrieval()
+        s2_retrieval = kaska.run_retrieval()
```

Comment:
If `s2_retrieval` is a namedtuple with `temporal_grid`, `laii` and so on, wouldn't it be simpler to just extract `parameter_names` from `list(s2_retrieval._asdict().keys())`? Or
```python
param_names, param_data = ([k for k in s2_data._asdict().keys()], [v for k, v in s2_data._asdict().items()])
```

Reply:
Implemented something similar to eliminate the extract_parameters() function.